hierarc/Likelihood/cosmo_likelihood.py

```diff
@@ -75,6 +76,8 @@ def __init__(
          according to the lens posterior kwargs 'lambda_scaling_property_beta'
         :param kappa_ext_sampling: bool, if True samples a global external convergence parameter
         :param kappa_ext_distribution: string, distribution function of the kappa_ext parameter
+        :param kappa_ext_alt: bool, if True samples kappa_ext with the kappa_ext_alt and/or kappa_ext_alt_sigma
```

Comment:
would it be possible to define a different kappa_ext_distribution instead of introducing a new parameter here? I don't understand the meaning of this extra parameter.